src/compiler/transformers/generators.ts

```diff
@@ -1124,6 +1124,18 @@ namespace ts {
                 //  .mark resumeLabel
                 //      _b.apply(_a, _c.concat([%sent%, 2]));
 
+                // The `ImportCall` Expression isn't really a call expression, and the import keyword cannot be removed from its arguments
+                if (isImportCall(node)) {

```

Comment:
I don't think https://github.com/Microsoft/TypeScript/pull/18998#discussion_r143319421 was addressed successfully:
> Actually, all you need to do is change line 1115 (https://github.com/Microsoft/TypeScript/pull/18998/files/5d5fa0d49afb9a4c3b36ed48c8961a10b7eaed6e#diff-7c8a35e1da8c32e5aa5fd22d015c443dR1115) to be `if (!isImportCall(node) && forEach(...)` and let the default behavior of `visitEachChild` below do all the work for you.

Reply:
Ah! You're right! (For some reason I thought this comment was on one of the other import call PRs.) Got it!